centos.org/jobs/katello-pipelines.yml

```diff
@@ -25,6 +25,7 @@
     distro:
       - centos7
       - centos8
```

Comment:
Will this still work? Shouldn't this be dropped or do we have hacks to provide it with a repository (like from vault)?